samples/data-ingestion/README.md

```diff
@@ -0,0 +1,108 @@
+---
+name: ChatGPT + Enterprise data + data ingestion
+description: Learn how to index your documents with the prepdocs script.
+languages:
+- python
+- typescript
+- bicep
+- azdeveloper
+products:
+- azure-openai
+- azure-cognitive-search
+- azure-app-service
+- azure
+page_type: sample
+urlFragment: azure-search-openai-demo-data-ingestion
+---
+# Indexing documents for the Chat App
+
```

Comment:
I feel like each of these needs some sort of preliminary paragraph so that developers aren't confused when they land up on them, as we didn't write them with the samples browser in mind.

Like this one:

"The [azure-search-openai-demo](https://github.com/Azure-Samples/azure-search-openai-demo/) project can set up a full RAG chat app on Azure AI Search and OpenAI so that you can chat on custom data, like internal enterprise data or domain-specific knowledge sets. For full instructions on setting up the project, consult the [main README], and then return here for detailed instructions on the data ingestion component."